l10n/de/browser/browser/menubar.ftl

```diff
@@ -18,7 +18,7 @@ menu-file-new-window =
     .label = ­Neues Fenster
     .accesskey = N
 menu-file-new-private-window =
-    .label = Neues privates Fenster
+    .label = Neues Vergessen Fenster
```

Comment:
@tsl143 I am not sure this is correct. Could you please check again with a German native speaker?

Reply:
Checked with Elisa, it's fine

Reply:
Thank you :)